clients/mtaclient/mta_rest_client.go

```diff
@@ -65,9 +66,10 @@ func (c MtaRestClient) GetMta(mtaID string) (*models.Mta, error) {
 	return result.(*operations.GetMtaOK).Payload, nil
 }
 
-func (c MtaRestClient) GetMtaFiles() ([]*models.FileMetadata, error) {
+func (c MtaRestClient) GetMtaFiles(namespace *string) ([]*models.FileMetadata, error) {
 	params := &operations.GetMtaFilesParams{
-		Context: context.TODO(),
+		Context:   context.TODO(),
+		Namespace: namespace,
```

Comment:
If you decide to go without the pointer, this code line would look like this:
Namespace: &namespace